### PR TITLE
Typo fixes. Move to uselagoon for images. A few linting issues.

### PR DESCRIPTION
**Pull request by seanhamlin**
This was after just importing the project into PHPStorm, and looking at the issues found.

### CODE REVIEW

File: src/Commands/sailonLagoonAssets/stubs/mariadb.stub [hunk 1]
```diff
@@ -1,5 +1,5 @@
   mariadb:
-    image: amazeeio/mariadb
+    image: uselagoon/mariadb
```
Comment (tobybellwood) [inline src/Commands/sailonLagoonAssets/stubs/mariadb.stub]:
```suggestion
    image: uselagoon/mariadb-10.11
```

File: src/Commands/sailonLagoonAssets/stubs/mysql.stub [hunk 2]
```diff
@@ -1,5 +1,5 @@
   mariadb:
-    image: amazeeio/mariadb
+    image: uselagoon/mariadb
```
Comment (tobybellwood) [inline src/Commands/sailonLagoonAssets/stubs/mysql.stub]:
```suggestion
    image: uselagoon/mysql-8.0
```

File: src/Commands/sailonLagoonAssets/stubs/redis.stub [hunk 3]
```diff
@@ -2,6 +2,6 @@
     build:
       context: .
       dockerfile: lagoon/redis.dockerfile
-    image: amazeeio/redis-persistent
+    image: uselagoon/redis-persistent
```
Comment (tobybellwood) [inline src/Commands/sailonLagoonAssets/stubs/redis.stub]:
```suggestion
    image: uselagoon/redis-7-persistent
```

File: tests/Unit/assets/stubs/mariadb.stub [hunk 4]
```diff
@@ -1,5 +1,5 @@
   mariadb:
-    image: amazeeio/mariadb
+    image: uselagoon/mariadb
```
Comment (tobybellwood) [inline tests/Unit/assets/stubs/mariadb.stub]:
```suggestion
    image: uselagoon/mariadb-10.11
```

File: tests/Unit/assets/stubs/mysql.stub [hunk 5]
```diff
@@ -1,5 +1,5 @@
   mariadb:
-    image: amazeeio/mariadb
+    image: uselagoon/mariadb
```
Comment (tobybellwood) [inline tests/Unit/assets/stubs/mysql.stub]:
```suggestion
    image: uselagoon/mysql-8.0
```

**Comment by tobybellwood** [comment]
Just smooshed some versioned images in here - maintaining the unversioned ones is a PITA - so trying to migrate people now!

File: tests/Unit/assets/stubs/mysql.stub (hunk 5 above)
Comment (seanhamlin) [reply tests/Unit/assets/stubs/mysql.stub]: I thought it was weird that the mysql stub was referencing mariadb

Comment (tobybellwood) [reply tests/Unit/assets/stubs/mysql.stub]: historically it had to, until we made MySQL images :partying_face: (and in non-drupal world, the creds are the same)

**Review by bomoko — APPROVED** [review]
Great stuff

**Comment by bomoko** [comment]
Thanks Chaps

**Pull request merged**